app/assets/stylesheets/application.css

```diff
@@ -7,3 +7,19 @@
   width: 100px;
   height: 100px;
 }
+
```

Comment:
- [ ] [OPTIONAL]
Let's ensure the `Create Category` button is sticky to the bottom of the mobile screen, like seen in the original design. Let's also ensure the same for the similar other pages.

**Your design:**
**Your splash screen:**     |  **Expected**     
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/M-Anwar-Hussaini/Expense-Management/assets/12745474/c8e8b78f-dccf-4764-b71a-0758295e464e"> | <img width="800" alt="Home Page" src="https://github.com/M-Anwar-Hussaini/Expense-Management/assets/12745474/b23a42bf-ee8c-46d6-8ca9-f0b762971127">